CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Download the dependencies using [`dep`](https://github.com/golang/dep).
 ```bash
 cd $GOPATH/src/github.com # Create this directory if it doesn't exist
 git clone git@github.com:<YOUR_FORK>/wave wave-k8s/wave
-dep ensure # Installs dependencies to vendor folder.
+go get -v ./cmd/manager/ # Download dependencies
```

Comment:
Can also do `go mod download` right? Is there a difference?

Reply:
I use `go get` here because `go mod download` is likely to modify the go.mod/go.sum files.

I see `go get -v` as enough (so that the user has their deps downloaded and can start) although if you think `go mod download` is the way I'm happy to update it

Reply:
OK I've updated it to `go mod download -x` (and tested)